hud.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python
```

Comment:
Why rename stratux_hud.py to hud.py?

Reply:
I didn't - I think that was the file that you accredited for the hud code.  I guess I inadvertently included that in the staging.  It can be removed

Reply:
Specifically, I went to his github and pulled it just to see what it was.

Reply:
... I was too focused on the name and the big blob of green to see that the code didn't really match.

Reply:
BTW - I think you need to push your new commit into this branch... and possibly change the PR to point to your new commit.